CHANGES.txt

```diff
@@ -1,4 +1,6 @@
 ﻿Current
+Fixed: GITHUB-1297: testng-failed.xml includes setup and tearDown (before and after annotations) of passed tests
+(Krishnan Mahadevan)
```

Comment:
@juherr  - So far in any of the PRs that I have raised I have not added the reporter name. That is why I didn't add it here. Moreover the reporter name can be easily found by viewing the bug report that is associated with a change in the Changes file :)